install/install.go

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package install
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/kubearmor/kubearmor-client/k8s"
+	"golang.org/x/mod/semver"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func K8sInstaller(c *k8s.Client) error {
+	env := autoDetectEnvironment(c)
+	if env == "none" {
+		return errors.New("unsupported environment or cluster not configured correctly")
+	}
+	fmt.Printf("Auto Detected Environment : %s\n", env)
+	fmt.Print("Service Account ...\n")
```

Comment:
Can we make these prints jazzy? For e.g., using emojis similar to what minikube, cilium clients do?
Sorry, I know this sounds a petty work but it has impact.
If you think this could be done later, that is fine too.

Reply:
Yes, I was totally planning to add those but if it's fine let's do it in a seperate PR?